src/main/java/com/opentable/jvm/Dcmd.java

```diff
@@ -41,7 +41,7 @@
 /**
  * Execute diagnostic commands.
  */
-class Dcmd {
+class Dcmd { //NOPMD
```

Comment:
utility class bs

Reply:
NOPMD is kind of a big hammer.  Another fix is adding the right kind of (never used) constructor, right?

Reply:
I prefer the `@SupressWarnings` to deal with PMD issues